### PR TITLE
Add an overview README for Salvo.

### PR DESCRIPTION
**Pull request by mum4k**

### CODE REVIEW

**Review by fei-deng — APPROVED** [review]
LGTM!

File: salvo-remote/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Contributions to Salvo on all levels, whether design, code or testing are
+welcome. Please contact us if you would like to participate.
+
+## Contact details
+
+### People
+
+- Project owner: Jakub Sobon ([mum4k](https://github.com/mum4k)) (mumak@google.com)
+- Contributor: Fei Deng ([fei-deng](https://github.com/fei-deng))
+- Contributor: [gyohuangxin](https://github.com/gyohuangxin)
```
Comment (gyohuangxin) [inline salvo-remote/README.md:60]: Can you also add my real name "Xin Huang", thanks!

Comment (mum4k) [reply salvo-remote/README.md]: Of course, done. Thank you for raising this, I did not want to make your name public without hearing from you first.

**Comment by mum4k** [comment]
I re-read the document and make a few sentences clearer, since I didn't do a good job distinguishing between the local and the remote modes before.

Also formatted the raw file by wrapping at 80 chars.

**Comment by mum4k** [comment]
@fei-deng @gyohuangxin Please take another look.

**Review by gyohuangxin — APPROVED** [review]
LGTM, thanks!

**Review by fei-deng — APPROVED** [review]
LGTM. Thanks for improving it.

**Pull request merged**